### PR TITLE
Feature/features on aftermath controller

### PR DESCRIPTION
**Pull request by zenangst**
This PR adds `enabledFeatures` to `AftermathController` so that you can enable and disable some of the `Spots` related features that you get with `AftermathController`.

### CODE REVIEW

File: Sources/AftermathController.swift [hunk 1]
```diff
@@ -64,7 +76,7 @@ public class AftermathController: SpotsController, CommandProducer {
 
   public override func viewDidLoad() {
     super.viewDidLoad()
-    spotsRefreshDelegate = self
+    toggle(features: enabledFeatures)
```
Comment (onmyway133) [inline Sources/AftermathController.swift:78]: Do we need to call `toggle` in both `init` and `viewDidLoad`?

Comment (vadymmarkov) [reply Sources/AftermathController.swift:78]: I guess we need it only here..Right @zenangst ?

Comment (zenangst) [reply Sources/AftermathController.swift:78]: Ah, this is true! I'll fix it :)

Comment (zenangst) [reply Sources/AftermathController.swift:78]: Should be fixed now

Comment (onmyway133) [reply Sources/AftermathController.swift:78]: @zenangst I can fix it.

Comment (zenangst) [reply Sources/AftermathController.swift:78]: @onmyway133 fix what?

**Comment by zenangst** [comment]
ping @hyperoslo/ios, anything more that we need to fix?

**Comment by vadymmarkov** [comment]
@zenangst 🚀

**Pull request merged**